packages/editor/package.json

```diff
@@ -75,7 +75,8 @@
 		"is-plain-object": "^5.0.0",
 		"memize": "^2.1.0",
 		"react-autosize-textarea": "^7.1.0",
-		"remove-accents": "^0.5.0"
+		"remove-accents": "^0.5.0",
+		"uuid": "^10.0.0"
```

Comment:
Most other packages are using `^9.0.1`, I wonder if we should stick to it for consistency and consider the upgrade in a separate PR.

Reply:
sure, will update

Reply:
updated in 7ac01caa54d5988fe6b65cc692c2b20504477510